src/ui/widgets/common/WidgetIcon.js

```diff
@@ -4,8 +4,10 @@ import cx from 'classnames';
 import { useSelector } from 'react-redux';
 
 import { getWidgetIcon } from 'state/widgets/selectors';
+import { getResourcePath } from 'helpers/microfrontends';
```

Comment:
`getResourcePath` is now not only used for MFE, so it makes sense to move it elsewhere

Reply:
moved to a resourcePath file